modules/perception/production/data/perception/camera/models/traffic_light_recognition/horizontal/deploy.prototxt

```diff
@@ -105,7 +105,7 @@ layer {
     stride_h: 2
     pad_w: 1
     pad_h: 1
-    round_mode: 1
+    cmp_out_shape_floor_as_conv: true
```

Comment:
1. I noticed that you haven't modify the corresponding `addPoolingLayer` function in `rt_net.cc`. Only if you specify this param by TensorRT IPoolingLayer's setPaddingMode(nvinfer1::PaddingMode::kCAFFE_ROUND_DOWN), can it be truly loaded. [https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_pooling_layer.html#ad0fabf28e72fb4c0f5ec5692444a79bf](url)
2. Therefore, there is no need to change the prototxt. Just keep all the round_mode usage unchanged. Instead, you should modify rt.proto, as Caffe already updated round_mode param in the proto: [https://github.com/BVLC/caffe/pull/6282/files](url)
3. The meaning of round_mode and TensorRT's corresponding param is explained here: [https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/namespacenvinfer1.html#a72f43f32e90e4ac5548f8c9ae007584c](url)

Reply:
And `cmp_out_shape_floor_as_conv` is only used to change the name of layer, it doesn't have any function.